source/remoteClient/cues.py

```diff
@@ -54,16 +54,16 @@ class Cue(TypedDict, total=False):
 
 def _playCue(cueName: str) -> None:
 	"""Helper function to play a cue by name"""
-	if not shouldPlaySounds():
+	if shouldPlaySounds():
+		# Play wave file
+		if wave := CUES[cueName].get("wave"):
+			nvwave.playWaveFile(os.path.join(globalVars.appDir, "waves", wave + ".wav"))
+	else:
 		# Play beep sequence
 		if beeps := CUES[cueName].get("beeps"):
 			filteredBeeps = [(freq, dur) for freq, dur in beeps if freq is not None]
 			beepSequenceAsync(*filteredBeeps)
```

Comment:
should this become an elif?

```suggestion
	elif beeps := CUES[cueName].get("beeps"):
		# Play beep sequence
		filteredBeeps = [(freq, dur) for freq, dur in beeps if freq is not None]
		beepSequenceAsync(*filteredBeeps)
```

Reply:
I think the symmetry of my version makes it easier to understand, but I am not bothered if you want to switch to an `elif`

Reply:
Going by zen of python, going with the "simplified" version